packages/e2e-tests/specs/editor/various/change-detection.test.js

```diff
@@ -302,14 +305,17 @@ describe( 'Change detection', () => {
 		// Keyboard shortcut Ctrl+S save.
 		await pressKeyWithModifier( 'primary', 'S' );
```

Comment:
If you insert `await page.evaluate( () => new Promise( window.requestIdleCallback ) );` after this line, the problem can be consistently reproduced, as you would expect. So if the test runs slow, the tests fails.